config/vufind/searchspecs.yaml

```diff
@@ -70,6 +70,63 @@
 #
 # ...etc.
 #
+# GlobalExtraParams allows extra parameters to be (conditionally) added to the Solr
+# search request:
+#
+# GlobalExtraParams:
```

Comment:
You have examples below with explanations, but you also have this unspecified example up here. Should this get a label too, or should it be made more generic so that it demonstrates all available syntax rather than looking like a real-world example?

Reply:
Done.